sass/style.scss

```diff
@@ -0,0 +1,250 @@
+body {
+  font-family: 'Montserrat', sans-serif;
+  color: #5c5c5c;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.29;
+
+  h1 {
+    font-size: 36px;
+    font-weight: 700;
+    line-height: 1;
+    color: #000;
+  }
+
+  h2 {
+    font-size: 26px;
+    font-weight: 700;
+    line-height: 1.14;
+    color: #000;
+  }
+
+  h3 {
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 1.23;
+    color: #000;
+  }
+
+  .button {
+    color: #fff;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 1.09;
+    background-color: #1b63f0;
```

Comment:
You should use variables for storing palette color.
Fix it please all over this file

Reply:
I must use variables for all colors on the page?